src/modules/BootSupportCheck.rb

```diff
@@ -191,6 +194,8 @@ def check_zipl_part
         return false
       end
 
```

Comment:
Ok. Reusing the logic from `Y2Storage::BootRequirementsChecker` sounds reasonable. But then this code should be substituted by a call to that checker, not completely commented out without even including a comment explaining why the code was commented instead of updated.

That being said. This logic does not match 100% with the logic implemented in `BootRequirementsChecker` as it is [documented here](https://github.com/yast/yast-storage-ng/blob/master/doc/boot-requirements.md#needed-partitions-in-a-s390-system). According to that, the checker always requires a separate extX partition mounted at `/boot/zipl`. But the code we just commented is fine if that partition is not completely separate, as long as the filesystem containing `/boot/zipl` is extX. We need to clarify what is more correct.